data_transfer/create_dataset_dump_for_release.py

```diff
@@ -259,7 +259,11 @@ def copy_vcf_to_release(dataset: str, billing_project: str | None):
     vcf_analyses = [
         analysis
         for analysis in analyses
-        if ('stage', 'DatasetVCF') in analysis['meta'].items()
+        if (
+            ('stage', 'DatasetVCF') in analysis['meta'].items()
+            or ('stage', 'AnnotatedDatasetMtToVcfWithHailQuery')
+            in analysis['meta'].items()
+        )
```

Comment:
```suggestion
        if analysis['meta'].get('stage') in ['DatasetVCF', 'AnnotatedDatasetMtToVcfWithHailQuery']
```

quick simplification suggestion